addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -952,8 +952,11 @@ registry.socialShare = publicWidget.Widget.extend({
     _renderSocial: function (social) {
         var url = this.$el.data('urlshare') || document.URL.split(/[?#]/)[0];
         url = encodeURIComponent(url);
-        var title = document.title.split(" | ")[0];  // get the page title without the company name
-        var hashtags = ' #' + document.title.split(" | ")[1].replace(' ', '') + ' ' + this.hashtags;  // company name without spaces (for hashtag)
+        const titleParts = document.title.split(" | ");
+        const title = titleParts[0]; // Get the page title without the company name
+        const hashtags = titleParts.length === 1
+            ? ` ${this.hashtags}`
+            : ` #${document.title.split(" | ")[1].replace(" ", "")} ${this.hashtags}`; // Company name without spaces (for hashtag)
```

Comment:
```suggestion
            // Company name without spaces (for hashtag)
            : ` #${titleParts[1].replace(" ", "")} ${this.hashtags}`;
```
Why respliting? Can use `titleParts` directly